tests/integration/components/personal-survey-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | survey-window', function(hooks) {
```

Comment:
Tal vez se podría testear los métodos que guardan las preguntas respondidas / saber la próxima.

Reply:
Perdí 1 hora intentando llamar a esos métodos desde los tests. Me rindo.

Reply:
En mi PR hice algo parecido: https://github.com/Program-AR/pilas-bloques/pull/671/files#diff-6e5574ce1b889730a79a0e272722be42

Reply:
Mirá, ví la documentación de Ember para testear service (lo de hacer ` this.owner.lookup('service:analytics-api')` ) pero en la de component ni hacía referencia a eso. ¿Vos decís que si hago ` this.owner.lookup('component:personal-survey')` hace la magia? En cuanto tenga un rato pruebo.

Reply:
Yo no leí nada, lo saqué de otros ejemplos de PB, así que ni idea :P